### PR TITLE
Events#index displays currently-running events

### PR DESCRIPTION
**Pull request by reshleman**
- Add class methods to `Event` to display current and closed events
- Show lists of current and closed events on `events#index`
- Update `_event` partial to show relative closing date
- Extract helper method for showing closing date relative to today

https://trello.com/c/amqWgDDf

### CODE REVIEW

File: app/helpers/date_helper.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+module DateHelper
+  def closing_date_in_words(date)
+    if date >= Date.today
```
Comment (hrs) [inline app/helpers/date_helper.rb:3]: A `still_open?(date)` predicate would read nicely here.

File: app/helpers/date_helper.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    when distance == -1
+      "yesterday"
+    when distance == 0
+      "today"
+    when distance == 1
+      "tomorrow"
+    when distance > 1
+      "in " + distance_of_time_in_words(from_date, to_date)
+    end
+  end
```
Comment (hrs) [inline app/helpers/date_helper.rb:27]: This is sad. We could do some elaborate polymorphism, I guess, but that's just beheading the hydra.

**Comment by hrs** [comment]
Apart from that predicate suggestion, :mailbox:

File: app/helpers/date_helper.rb (hunk 2 above)
Comment (reshleman) [inline app/helpers/date_helper.rb:27]: Yeah, I'm not thrilled about it, but it seems like the better alternative. I think it's _reasonably_ readable.

File: app/helpers/date_helper.rb (hunk 1 above)
Comment (reshleman) [inline app/helpers/date_helper.rb:3]: Good suggestion. Done. :smile_cat:

**Pull request closed without merging**